test/fixtures/rules/std_rule_cases/L063.yml

```diff
@@ -0,0 +1,111 @@
+rule: L063
+
+test_pass_basic:
+  pass_str: |
+    select foo from bar;
+
+test_fail_basic:
+  fail_str: |
+    select foo from bar;;
+  fix_str: |
+    select foo from bar;
+
+test_fail_newlines:
+  fail_str: "select foo from bar;\n;"
+  fix_str: "select foo from bar;\n"
+
+test_fail_block_comment:
+  fail_str: "select foo from bar;/*jpy-git is cool*/\n;"
+  fix_str: "select foo from bar;/*jpy-git is cool*/\n"
```

Comment:
This one is interesting I don't think empty statements are valid SQL with most engines. So neither pre nor post fix would this run.